### DISCUSSION TITLE
Is it possible to create candlestick indicators?

### DISCUSSION THREAD

**Discussion by asfand987** [klinecharts/KLineChart#628] [Q&A] [answered]
If anyone has been able to create any would they care to let me know how?

Unlike normal indicators candlestick indicators don't necessarily have a value for each data point. So I don't think that will work with current indicator View.

**Comment by char101** [comment, marked as answer]
Why not? 

Easy way just set your figure type to circle, for candles which have no value set the indicator value to `null`, for those that have values set the indicator value to candle `high`/`low`. Use the `styles` property to colorize the circles.

Hard way will be to draw your own indicator. Just skip the values that are null.
